src/install.rst

```diff
@@ -50,16 +50,18 @@ Install Nextstrain CLI
 
    .. group-tab:: Windows (WSL)
 
-      `Install Windows Subsystem for Linux (WSL) 2 <https://docs.microsoft.com/en-us/windows/wsl/install>`_.
+      1. `Install Windows Subsystem for Linux (WSL) 2 <https://docs.microsoft.com/en-us/windows/wsl/install>`_.
       You may have to restart your machine when configuring WSL.
 
+      1. Open a WSL terminal by clicking the Start menu, typing ``wsl``, and pressing enter.
+
       In a WSL terminal, run:
 
       .. code-block:: bash
 
          curl -fsSL --proto '=https' https://nextstrain.org/cli/installer/linux | bash
 
-      You can launch a WSL terminal by clicking the Start menu, typing ``wsl``, and pressing enter.
+      You can launch a WSL terminal by searching for ``wsl`` in the Windows Start menu and opening the result that says **wsl: Run command**.
```

Comment:
_non-blocking_

Consider using the `guilabel` role here (and in parallel places) instead of bold face:

```suggestion
      You can launch a WSL terminal by searching for ``wsl`` in the Windows Start menu and opening the result that says :guilabel`wsl: Run command`.
```

Reply:
Done: c5a2129